certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/README.md

```diff
@@ -1,24 +1,23 @@
-# IN-D Insurance (ICD10/CPT)
+# IN-D Insurance (ICD10/CPT autocoding)
```

Comment:
@gayathri35 The title should be changed everywhere in the connector artifacts.

Reply:
Hi Srikanth,
I have changed the title: IN-D Insurance (ICD10 & CPT) everywhere in the connector artifacts. Please check once. Sorry for the inconvenience.